src/test/pmem2_config_get_file_size/TESTS.py

```diff
@@ -64,6 +65,7 @@ def run(self, ctx):
 
 # "open" fails for directories
 @t.windows_exclude
+@t.require_granularity(t.ANY)
```

Comment:
What's the default? Shouldn't it be ANY?

BTW, do we really have globally defined word `ANY` in the `t` namespace?